android/app/build.gradle

```diff
@@ -84,6 +84,7 @@ dependencies {
     compile 'com.squareup:otto:1.3.8'
     compile 'com.jakewharton.timber:timber:4.1.0'
     compile 'com.jakewharton:butterknife:7.0.1'
+    compile 'com.github.florent37:picassopalette:2.0.0'
```

Comment:
No need to add external libraries. Please use the Pallette provided in AppCompat